collectd-mlab.spec

```diff
@@ -130,6 +179,14 @@ fi
 chkconfig crond on
 chkconfig rsyslog on
 chkconfig collectd on
+chkconfig collectd-setup on
+
+# TODO(soltesz): fix mlab_export.py to handle initial conditions correctly.
+# Initialize the lastexport timestamp file to one hour ago.
+touch -t $( date +%Y%m%d%H00 -d "-1 hour" ) /var/lib/collectd/lastexport.tstamp
+
+# TODO(soltesz): what package should own this file?
+touch %{site_packages}/mlab/__init__.py
```

Comment:
The file `mlab/disco/__init__.py` is declared in the `%files` section above.

This file `mlab/__init__.py` is needed so that python scripts can import the `mlab.disco` module, For example:

    from mlab.disco import network

But, I put this `touch` here because I didn't think that collectd-mlab should own the `mlab` root directory. The TODO is to find a better owner. I'm thinking the `package` repo should own this, but then the collectd-mlab package has a dependency on it in a weird way. I don't have a better solution yet.